packages/lit-dev-content/site/home/1-splash.html

```diff
@@ -1,5 +1,5 @@
 <section id="new-banner">
-  <p>📣 Check out our brand-new <a href="{{site.baseurl}}/tutorials">Tutorials</a> page and <a href="{{site.baseurl}}/youtube">Lit YouTube Channel!</a></p>
+  <p>📣 Check out our brand-new <a href="{{site.baseurl}}/discord/">Discord server!</a></p>
```

Comment:
I've felt that as rendered, the space between the megaphone and the first word is a few px too crowded... anyone else?

Reply:
the megaphone on mac isn't very legible anyway due to contrast 🤷‍♂️

Reply:
I made some tweaks to the banner, PTAL!

![image](https://user-images.githubusercontent.com/48894/187810638-64e7fed7-9fe6-4ddc-8f52-ee151bd8ca10.png)


- Used the darker Lit blue for the background. Improves contrast. The text is kinda on the lighter blue is kinda hard for me to read, as well as the emoji not being super legible.

- Increased font size. Decreased padding.